deploy/kubernetes/sequencer-deployment.yaml

```diff
@@ -27,9 +27,19 @@ spec:
         - --alsologtostderr
         - --v=5
         image: us.gcr.io/key-transparency/keytransparency-sequencer:latest
+        livenessProbe:
```

Comment:
Would this capture what you're trying to do a bit more concisely?
 ```
livenessProbe:
          httpGet:
            path: /metrics
            port: 8081
```

Reply:
Yes, that's probably the more efficient, kubernets specific, way of doing it. 
However, in an effort to only maintain one set of deployment configs, I'm auto generating the kubernetes configs with [`kompose`](https://github.com/google/keytransparency/blob/master/deploy/kubernetes/gen.go) from the `docker-compose.yml` file, and docker-compose uses a `cmd` format for liveness checks.

Reply:
Makes sense, though I suspect you're going to have to diverge at some point for a more nuanced kubernetes config, but no need to burden yourself with that now!

Reply:
Yep. I'll be diverging now. 

docker-compose doesn't support the side-car strategy that prometheus-to-sd requires.

Reply:
:)

Reply:
Will you be changing the `livenessProbe` too then?

Reply:
done